Plugin.cs

```diff
@@ -106,7 +106,7 @@ public IEnumerator GetSpriteFromPath(string path)
         
         public IEnumerator Request(string path)
         {
-            path = "file:\\\\"+Path.GetFullPath(path);
+            path = "file://"Path.GetFullPath(path);

```

Comment:
There's a small typo on line 109, should be:
```suggestion
            path = "file://" + Path.GetFullPath(path);
```